lib/volabit/api/transactions.rb

```diff
@@ -12,16 +12,16 @@ def buy_bitcoins(amount:)
   # Instantly sell bitcoins to get fiat balance.
   # @note The amount is expected in satoshis.
   def sell_bitcoins(amount:)
-    post_to_resource 'api/v1/users/me/sells', {
+    resource :post, 'api/v1/users/me/sells', {
       amount: amount
     }
   end
 
   # Instantly send fiat or bitcoins to an address.
-  # @note The amount is expected in satoshis for bitcoins and cents for fiat
+  # @note The amount is expected in satoshis for bitcoins and cents for
   #       fiat currencies.
-  def send(currency:, amount:, address:)
-    post_to_resource 'api/v1/users/me/send', {
+  def send_money(currency:, amount:, address:)
```

Comment:
Yes, but... I can't add the alias `send` for `send_money`, because it clashes with `Object#send`. So, I will clarify the change in the change log and the release notes to just bump the version to 1.1.0.